src/tests/issues.targets

```diff
@@ -1274,6 +1274,19 @@
             <Issue>Crashes during LLVM AOT compilation.</Issue>
         </ExcludeList>
 
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/Methodical/Methodical_d1/**">
+            <Issue>Crashes during LLVM AOT compilation.</Issue>
+        </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/Methodical/Methodical_do/**">
+            <Issue>Crashes during LLVM AOT compilation.</Issue>
+        </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/Methodical/Methodical_r1/**">
+            <Issue>Crashes during LLVM AOT compilation.</Issue>
+        </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/Methodical/Methodical_ro/**">
+            <Issue>Crashes during LLVM AOT compilation.</Issue>
+        </ExcludeList>
```

Comment:
we need to file an issue for these

Reply:
do you remember where did it fail?

Reply:
Interestingly, it's failing on fullAOT-mini lane as well https://github.com/dotnet/runtime/issues/105395.

Reply:
Also, why is it still failing even after disabling the test for all Mono... https://dev.azure.com/dnceng-public/public/_build/results?buildId=753237&view=logs&j=9c845561-93d9-5f12-0979-955ea2f35497&t=644e769d-37f3-54b4-c202-c63c42bf5d0c

Reply:
Apparently this is an incorrect way to disable tests with aot. I used to get failures with these tests on llvm fix PR https://github.com/dotnet/runtime/pull/105867. Testing locally there were failures for missing aot image because the tests where not being aot compiled but we still tried to run them. Together with the other llvm fixes, enabling these back fixed the failures.